### ISSUE TITLE
ООО "Автосалон Глобус"

### ISSUE THREAD

**Issue by Shtukin** [ashustov/test#43]
У Владимира

**Comment by Shtukin** [comment]
отправили дилеру, ждём внедрения

**Comment by Shtukin** [comment]
должны дать удалённый доступ к 1C

**Comment by Shtukin** [comment]
Колесник Сергей	79202377190

**Comment by vvveryaskin** [comment]
Есть замечания по классификации по виду ремонта: дополнительное оборудование. Принято решение доработать конфигурацию по классификации цехов как в CRM с делением на виды ремонта и цеха. Контакт запланирован 21.08 в 15:00

**Comment by Shtukin** [comment]
@vvveryaskin  Обрати еще внимание на коды запчастей http://joxi.ru/Vm6OByBfDqJyq2 это очень важный момент. То что сейчас вижу не соответствует кодам запчастей УАЗ.  Они должны быть такими http://joxi.ru/Drl8XaXivQBMqr

**Comment by vvveryaskin** [comment]
Да, хорошо. Я так понимаю просто артикулы (каталожные номера) запчастей туда нужно передавать.

**Comment by Shtukin** [comment]
да

**Comment by vvveryaskin** [comment]
Связался с клиентом. Подключился по удаленке внес изменения в конфигурацию:
1) добавил связку поста не только к цехам, но и видам ремонта (дополнительное оборудование)
2) В качестве кода запчасти и работы будет передаваться артикул

**Comment by Shtukin** [comment]
мало заказ-нарядов, нет возможности проверить корректность

**Comment by vvveryaskin** [comment]
Что с моей стороны необходимо сделать?

**Comment by vvveryaskin** [comment]
Во всех них придется заполнять обязательные реквизиты: вид ремонта по crm  и причина обращения (ее они тоже редко указывают). Но  там такой момент, что у них стоит запрет на редактирование закрытых заказ нарядов, по крайней мере для той учетки, под которой мне давали зайти в базу

**Comment by Shtukin** [comment]
Связался с Колесник, должен проверить заполнение обязательных полей.

**Comment by Shtukin** [comment]
решили оставить как есть до середины сентября, т.к. поток машин маленький. Когда вернёшься из отпуска проверим еще раз и выгрузим уже на боевую.

**Comment by vvveryaskin** [comment]
Связался с клиентом, подключился по удаленке. Сменил настройки подключения на https://crm.uaz.ru
api@dealer424.ru
testtest

**Comment by Shtukin** [comment]
нет мастера приёмщика http://joxi.ru/L21KELEh8y9K7r

**Comment by vvveryaskin** [comment]
мм, должность может иметь другой id в тестовой и рабочей црм?

**Comment by Shtukin** [comment]
хороший вопрос, сейчас проверим

**Comment by Shtukin** [comment]
В программе может быть несколько должностей с различными названиями, которые будут иметь системный код ACCEPTOR - это все "мастера-приемщики" для программы, т.к. она ориентируется именно на системный код. 
Дилеру необходимо найти должность по системному коду, и, если их несколько, то выбрать подходящую по наименованию. И уже id этой должности сопоставить в 1С.
При этом id должностей в рабочей и тестовой системах могут отличаться.

**Comment by vvveryaskin** [comment]
думаю это я не догадался. Надо в 1С регистр с ид из црм зачистить. ЦРМ сменилась, а ид в 1С хранятся из старой. Вот он и не находит по ним объект в новой ЦРМ

**Comment by Shtukin** [comment]
выгрузка не работает, за сентябрь всего один заказ-наряд http://joxi.ru/1A53BbBcnREgWm

**Comment by vvveryaskin** [comment]
Назначил контакт в 15:00

**Comment by vvveryaskin** [comment]
Исправлено: были расхождения в ID цеха. Как оказалось в тестовой https://uaz.autocrm.ru и рабочей https://crm.uaz.ru  разные ID цеха. В 1С нужно было заменить этот ID. Попробовал провести пару заказ нарядов - данные были отправлены.

**Comment by Shtukin** [comment]
косяк в видах ремонта, нужно сопоставить с "техническим обслуживанием" http://joxi.ru/Vm6OByBfD5Pej2

**Comment by vvveryaskin** [comment]
такое возможно только если они старый справочник виды ремонта по acrm оставили. Буду связываться

**Comment by Shtukin** [comment]
продолжают идти ЗН с не корректными видами ремонта http://joxi.ru/l2Z59v9swG41Nm

**Comment by vvveryaskin** [comment]
Согласован контакт на сегодня

**Comment by vvveryaskin** [comment]
http://joxi.ru/krDgkLgiE55WbA

**Comment by Shtukin** [comment]
оплачена
